homeassistant/components/myuplink/quality_scale.yaml

```diff
@@ -61,12 +61,12 @@ rules:
     comment: |
       Not possible to discover these devices.
   docs-data-update: done
-  docs-examples: todo
+  docs-examples: done
```

Comment:
this one is not done yet imo. `docs-examples` means actual ready to use automation examples in terms of YAML code or blueprints.

Reply:
Added an explicit example automation. 
Please note that available entites and entity names varies between heat-pump models and firmware versions, and it is therefore difficult to create an example automation that will work for most users. This one will show as an example of the principals.
https://www.home-assistant.io/integrations/myuplink/#example